README.md

```diff
@@ -45,23 +45,23 @@ WGER_API_KEY=123456
 ```
 
 To just run/develop the app it only needs to have any value for WGER_API_KEY, but
-you need a correct value if you want to register via the app. For this you need
-to allow (a probably dedicated) user on the wger server to register users in its
+you need a correct value if you want to register via the app. For this, you need
+to allow (a probably dedicated) user on the wger server to register users on its
 behalf. For this, generate an API KEY by visiting <http://localhost:8000/de/user/api-key>
-on your local instance and then run ``python3 manage.py add-user-rest theusername``
+on your local instance and then run ``python3 manage.py add-user-rest the username``
 
-You can later list all the registered users with: ``python3 manage.py list-users-api``  
+You can later list all the registered users with: ``python3 manage.py list-users-API``  
```

Comment:
`list-users-API` is the name of the python script, I'm not sure whether calling it is case insensitive